GitSocial/GitSocial/GSButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  GSButton.swift
+//  GitSocial
+//
+//  Created by Dillon on 11/27/21.
+//
+
+import UIKit
+
+class GSButton: UIButton {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    /// Required init used with storyboards
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    /// Custom init for our button
+    init(backgroundColor: UIColor, title: String) {
```

Comment:
We may want to update this init to use a default value of "Ok" for the button title, so that we don't need to do so at callsite
```suggestion
    init(backgroundColor: UIColor, title: String? = "Ok") {
```

Reply:
Opened issue #4